ansible/roles/accumulo/templates/log4j-service.properties

```
@@ -0,0 +1,24 @@
+log4j.appender.file=org.apache.log4j.RollingFileAppender
+log4j.appender.file.File=${accumulo.log.dir}/${accumulo.application}.log
+log4j.appender.file.MaxFileSize=1000MB
+log4j.appender.file.MaxBackupIndex=10
+log4j.appender.file.Threshold=ALL
+log4j.appender.file.layout=org.apache.log4j.PatternLayout
+log4j.appender.file.layout.ConversionPattern=%d{ISO8601} [%-8c{2}] %-5p: %m%n
+
+# send logs to Accumulo monitor
+log4j.appender.monitor=org.apache.accumulo.monitor.util.AccumuloMonitorAppender
+log4j.appender.monitor.Threshold=WARN
+
+# suppress spammy logging from 3rd party libs
+log4j.logger.org.mortbay.log=WARN
+log4j.logger.org.apache.zookeeper=ERROR
+log4j.logger.org.apache.hadoop.io.compress=WARNi
+
+log4j.logger.org.apache.accumulo=DEBUG
+
+# suppress some spammy Accumulo logging
+log4j.logger.org.apache.accumulo.core.file.rfile.bcfile=INFO
+log4j.logger.org.apache.accumulo.audit=OFF
+
+log4j.rootLogger=INFO, file
```

I think you need to lower this to DEBUG also, otherwise child loggers won't get the DEBUG logs.
```suggestion
log4j.rootLogger=DEBUG, file
```

---

I did this at first and I was getting too much junk from non-accumulo stuff (like GSon).  So that is when I decided to set the root INFO and `org.apache.accumulo` to DEBUG.

---

I'm surprised that worked. I think I need to re-learn how log4j configuration works.

---

Meta question.  The github commit button for your suggested change is grayed out now, did you explicitly do something to make that happen?  Just curious if that happened because you resolved the conversation.

---

Not sure. I didn't intentionally do anything that would've affected that.